icons/icons.json

```diff
@@ -2594,6 +2598,30 @@
 			}
 		]
 	},
+	"file_type_python_typed": {
+		"aliases": [
+			{
+				"extensions": [
+					"py.typed"
+				],
+				"name": "Plain Text (Python)",
+				"scope": "text.plain.python"
+			}
+		],
+		"color": "blue"
+	},
+	"file_type_python_version": {
+		"aliases": [
+			{
+				"extensions": [
+					".python-version"
+				],
+				"name": "Plain Text (Python)",
+				"scope": "text.plain.python"
```

Comment:
Two definitions with same scope doesn't work. Also do we really need dedicated icons for every python related meta-data file?

Reply:
I certainly do not think so, but I wouldn't consider these "every python related meta-data file" - just the bare minimum expected in a reasonably-sized Python project **and** tied to the tools included with the runtime.

Reply:
Changed to extend `source.python` instead.